include/gepetto/gui/fwd.hh

```diff
@@ -19,8 +19,12 @@
 
 #include <vector>
 
-#include <QtGui/qopengl.h>
+#include <QtGlobal>
 #include <QtGui>
+#if QT_VERSION >= 0x050000
+    #include <QtGui/qopengl.h>
+#endif
```

Comment:
@jcarpent does the order matter here ?
I remember that it Qt5 and OpenGL includes are a bit touchy.

Reply:
`#include <QtGui>`should be included before `#include <QtGui/qopengl.h>`

Reply:
But you did the opposite here: https://github.com/Gepetto/gepetto-viewer-corba/commit/3dc7f167ec9c7e3bca6c54180bf153dbdeb5b613

Reply:
I meant the reverse, sory.

Reply:
Ok, thanks.